content/subscription/core-subscription/remove-seats.md

```diff
@@ -29,4 +30,4 @@ For example, if you receive your billing on the 8th of every month for 10 seats
 
 5. The **Billing** tab confirms the seat removal and the details on when the changes take effect.
 
-To cancel the removal of seats, [contact support](https://hub.docker.com/support/contact/).
\ No newline at end of file
+You can cancel the removal of seats before your next billing cycle. To do so, select **Cancel change** in the bottom-right corner.
```

Comment:
might be worth highlighting that the change is scheduled to take effect at the end of the current billing cycle

Reply:
We have a little blurb about this at the top of the doc but isn't in the diff 👍